install/ks/centos7/ksintegralstor.cfg

```diff
@@ -599,18 +599,17 @@ else
   echo "Non dell hardware. Exiting..."
 fi
 
-### Grub and other file modification to show title/name Integralstor Unicell instead Centos 7.2 ###
+### Grub and other file modification to show title/name Integralstor instead Centos 7.2 ###
 cd /etc
-rm -f issue centos-release 
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/issue
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/centos-release
-cd /usr/share/plymouth/themes/text
-rm -f text.plymouth
-/usr/bin/wget -c http://192.168.1.150/netboot/distros/centos/6.6/x86_64/integralstor/v1.0/text.plymouth
-dracut -f
+mv issue issue.bak
```

Comment:
Its used for displaying console login menu text before prompting login. Like, by default in console we see os and kernel version displayed, under its asking for login.
Now after we changed console login method for users to access console options we have entries like;
^[[3;J^[[H^[[2J
Welcome to ^[[0;32;49mIntegralSTOR

^[[0;39;49mDate: ^[[0;36;49m\d \t
^[[0;39;49mNode: ^[[0;36;49m\n ^[[0;36;49m\s \r
^[[0;39;49m
To acces the menu, please login.

For console user to login before he get access to console options.